pkg/ddevapp/ddevapp.go

```diff
@@ -1619,21 +1618,31 @@ func (app *DdevApp) ListSnapshots() ([]string, error) {
 		return snapshots, nil
 	}
 
-	files, err := ioutil.ReadDir(snapshotDir)
+	dirEntrySlice, err := os.ReadDir(snapshotDir)
 	if err != nil {
 		return snapshots, err
 	}
 
+	// Get the mod times aligned with the DirEntry slice for comparison below.
+	dirEntryModTimeSlice := make([]time.Time, len(dirEntrySlice))
```

Comment:
I'm open to other ideas on how this can be handled. The basic issue is that `DirEntry.Info()` has a two-value return, so `ModTime()` can't be accessed directly.